.github/workflows/release.yml

```
@@ -62,12 +66,24 @@ jobs:
         if: ${{ github.event_name == 'pull_request' }}
       - run: 'echo -e "env_variables:\n  ENV_NAME: \"prd\"" >> app.yaml'
         if: ${{ github.event_name == 'push' }}
+      - uses: google-github-actions/setup-gcloud@v0.2.1
+        with:
+          project_id: 'hato-atama'
+      - id: 'auth'
+        name: 'Authenticate to GCP'
+        uses: google-github-actions/auth@v0.3.1
+        with:
+          create_credentials_file: 'true'
+          workload_identity_provider: 'projects/765091727073/locations/global/workloadIdentityPools/hato-atama-workload-identity/providers/github'
+          service_account: 'actions-deploy@hato-atama.iam.gserviceaccount.com'
+      - id: 'gcloud'
+        name: 'gcloud'
```

```suggestion
        name: 'Login to Google Cloud'
```

ログインしていることがわかるようなステップ名の方が良さそう。